lib/widgets/plan_widget.dart

```diff
@@ -0,0 +1,159 @@
+        child: IntrinsicHeight(
+          child:
+              Row(mainAxisAlignment: MainAxisAlignment.spaceBetween, children: <
+                  Widget>[
+            Flexible(
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Padding(
+                      padding:
+                          EdgeInsets.all(AppBoxProperties.containerPadding),
+                      child: Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: <Widget>[
+                            Text(
+                              this.planName,
+                              style: Theme.of(context).textTheme.headline2,
+                              overflow: TextOverflow.ellipsis,
+                              maxLines: titleMaxLines,
+                            ),
+                            Padding(
+                                padding: EdgeInsets.symmetric(
+                                    vertical:
+                                        AppBoxProperties.columnChildrenPadding),
+                                child: Text(
+                                  this.details,
+                                  style: Theme.of(context)
+                                      .textTheme
+                                      .subtitle1
+                                      .copyWith(
+                                          color:
+                                              chooseDetailsTextColor(context)),
+                                )),
+                            Row(
+                              crossAxisAlignment: CrossAxisAlignment.start,
+                              children: <Widget>[
+                                Padding(
+                                    padding: EdgeInsets.only(
+                                        right:
+                                            AppBoxProperties.buttonIconPadding),
+                                    child: Icon(
+                                      Icons.speaker_notes,
+                                      color: Colors.blue,
+                                      size: tasksIconSize,
+                                    )),
+                                Text(
+                                    ' ${AppLocales.of(context).translate("page.childSection.panel.content.tasks", {
+                                      'NUM_TASKS': this.tasksCount
+                                    })}',
+                                    style: blueTextStyle)
+                              ],
+                            )
+                          ])),
+                  Row(
+                    children: <Widget>[
+                      Expanded(child: addProgressIndicatorIfAvailable())
+                    ],
+                  )
+                ],
+              ),
+            ),
+            ClipRRect(
```

Comment:
This ClipRect is not needed. FlatButton has its own shape param that can take RoundedRectangleBorder, same with the other button